### ISSUE TITLE
HAT not compatible with official RPi enclosure

### ISSUE THREAD

**Issue by AnttiLukats** [raspberrypi/hats#34]
is there is new HAT spefication? Because any HAT design to the old one would not fit into the RPi official enclosure :(

**Comment by RandomlyKnighted** [comment]
I haven't heard of any new HAT specification. As far as I know the official Raspberry Pi enclosure is meant to hold only the Pi.

**Comment by medo64** [comment]
Is it allowable to make additional cutouts on the HAT that would allow it to fit into the official case; i.e. would board with additional cutouts on side still qualify as HAT?

Strictly reading specification, it would seem no as it does say "board must be 65x56 mm as per this drawing"

**Comment by RandomlyKnighted** [comment]
@medo64 Well technically the board is still 65x56 if it has a cutout in it, because the sides full length would meet the specs.

**Comment by medo64** [comment]
@RandomlyKnighted True, although part "per this drawing" part muddies the waters a bit. :)
